components/Cards/examples/supplemental/CardEditReorderCollectionCell.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  CardEditReorderCollectionCell.swift
+//  CatalogByConvention
+//
+//  Created by Galia Kaufman on 7/6/18.
+//
+//  Card images source:
+//  Material Design Guidelines (https://material.io/design/components/cards.html#card-collections)
+//
+
+import UIKit
+import MaterialComponents.MaterialCards
+
+class CardEditReorderCollectionCell: MDCCardCollectionCell {
+
+  var colorScheme = MDCSemanticColorScheme()
+  var typographyScheme = MDCTypographyScheme()
+
+  private lazy var imageView: UIImageView = {
+    let imageView = UIImageView(frame: CGRect.zero)
+    imageView.contentMode = .scaleAspectFit
+    imageView.layer.cornerRadius = 6.0
+    imageView.translatesAutoresizingMaskIntoConstraints = false
+    return imageView
+  }()
+
+  private lazy var titleLabel: UILabel = {
+    let label = UILabel(frame: CGRect.zero)
+    label.numberOfLines = 1
+    label.textAlignment = .center
+    label.lineBreakMode = .byTruncatingTail
+    label.font = self.typographyScheme.caption
+    label.translatesAutoresizingMaskIntoConstraints = false
+    return label
+  }()
+
+  func configure(title: String, number: Int) {
+
+    let bundle = Bundle(for: CardEditReorderCollectionCell.self)
+
+    let img = UIImage(named: "check-circle-blue", in: bundle, compatibleWith: nil)
+    self.setImage(img, for: .selected)
+
+    let maxImageNumber = 7
+    let imageName = "cell-image-\(number % maxImageNumber)"
+    self.imageView.image = UIImage(named: imageName, in: bundle, compatibleWith: nil)
+    self.titleLabel.text = title
+
+    self.insertSubview(imageView, at: 0)
+    self.insertSubview(titleLabel, at: 0)
+
+    titleLabel.setContentCompressionResistancePriority(800, for: .vertical)
+
+    if #available(iOS 11, *) {
+      post_iOS11Constraints()
+    } else {
+      pre_iOS11Constraints()
+    }
+  }
+
+  @available(iOS 11, *)
+  private func post_iOS11Constraints() {
+    #if swift(>=3.2)
+      let guide = self.safeAreaLayoutGuide
+      let margin: CGFloat = 8.0
```

Comment:
Maybe make these margins into constants used across both functions?

Reply:
It will be easier to remove the pre-ios11 code when we drop support - if the code is completely separate.

Reply:
As this is sample code, I'm not inclined to block landing the PR on this.